docs/storage/petalibrary/data_transfer/rclone.md

```diff
@@ -22,7 +22,7 @@ You must have administrative privileges on your laptop or lab server in order to
 
 ## Configure rclone
 
-The rclone application will require you to configure endpoints. Once your endpoints are configured, you can copy data to/from your local system to configured endpoints. Please note that rclone should only ever be connected to an RC Data Transfer Node (DTN). Because of this, we will configure an sftp endpoint in rclone that points to RC's DTN hosts. You must be connected to [CU's Network](https://oit.colorado.edu/services/network-internet-services/vpn) for this connection to work. For more information on DTNs, [check out our documentation on data transfers.](../../compute/data-transfer.md)
+The rclone application will require you to configure endpoints. Once your endpoints are configured, you can copy data to/from your local system to configured endpoints. Please note that rclone should only ever be connected to an RC Data Transfer Node (DTN). Because of this, we will configure an sftp endpoint in rclone that points to RC's DTN hosts. You must be connected to [CU's Network](https://oit.colorado.edu/services/network-internet-services/vpn) for this connection to work. For more information on DTNs, [check out our documentation on data transfers.](../../../compute/data-transfer.md)
 
 
```

Comment:
Please make the crazy long output in the code block below "Windows: From the Command Prompt, navagate to rclone file location and run: $ .\rclone.exe config" into a dropdown. You can put a code block in a dropdown.